tests/end-to-end/external-solution-integration/resourceskit/k8s_resources_check.go

```diff
@@ -0,0 +1,67 @@
+package resourceskit
+
+import (
+	"fmt"
+	"github.com/kyma-project/kyma/tests/end-to-end/external-solution-integration/consts"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	ingressNameFormat                 = "%s-application"
+	applicationProxyDeploymentFormat  = "%s-application-proxy"
+	applicationProxyRoleFormat        = "%s-application-proxy-role"
+	applicationProxyRoleBindingFormat = "%s-application-proxy-rolebinding"
+	applicationProxySvcFormat         = "%s-application-proxy-external-api"
+	eventServiceDeploymentFormat      = "%s-event-service"
+	eventServiceSvcFormat             = "%s-event-service-external-api"
+)
+
+type k8sResource struct {
+	name        string
+	kind        string
+	getFunction func(string, v1.GetOptions) (interface{}, error)
+}
+
+func newResource(name string, kind string, getFunc func(string, v1.GetOptions) (interface{}, error)) k8sResource {
+	return k8sResource{
+		name:        name,
+		kind:        kind,
+		getFunction: getFunc,
+	}
+}
+
+type K8sResourceChecker struct {
```

Comment:
I cannot find a place where this is used. Remove if unused.

Reply:
removed.